### PR TITLE
Add bulk adding users through AddFaceActivity

### PR DESCRIPTION
**Pull request by Etav99**

### CODE REVIEW

**Comment by Etav99** [comment]
[Masowe dodawanie użytkowników do bazy (dane testowe)](https://trello.com/c/Y1DkkmeH/28-masowe-dodawanie-u%C5%BCytkownik%C3%B3w-do-bazy-dane-testowe)

File: app/src/main/java/com/activities/AddFaceActivity.java [hunk 1]
```diff
@@ -283,7 +295,7 @@ private Bitmap preProcessFace(Bitmap image) {
     /**
      * Convert detected face to bitmap and display it on face image View.
      *
-     * @param face detected face.
+     * @param face - Detected face.
```
Comment (Qubaef) [inline app/src/main/java/com/activities/AddFaceActivity.java:298]: Dashes are not good idea in Javadocs. In autogenerated documentation there is dash added on default and adding dash in here will result in something like this: 
image

File: app/src/main/java/com/activities/AddFaceActivity.java [hunk 2]
```diff
@@ -299,7 +311,7 @@ public void run() {
     /**
      * Process face image using neural model and enable add user button when done.
      *
-     * @param face face image.
+     * @param face - Face image.
```
Comment (Qubaef) [inline app/src/main/java/com/activities/AddFaceActivity.java:314]: Same: dash

File: app/src/main/java/com/activities/AddFaceActivity.java [hunk 3]
```diff
@@ -312,12 +324,66 @@ private void processFace(Bitmap face) {
     /**
      * Start CameraActivity with in photo taking mode.
      *
-     * @param view - current view.
+     * @param view - Current view.
```
Comment (Qubaef) [inline app/src/main/java/com/activities/AddFaceActivity.java:327]: Same: dash

**Review by Qubaef — COMMENTED** [review]
I have tested the feature and it throws exception when there is wrong number of faces if the image:
image
Can't wy catch this error and ignore images with no faces? It will make things easier, cause right now we don't know if face detection will detect the face.

**Review by Qubaef — APPROVED** [review]

**Pull request merged**